Allow SubNav links' focus color to be set (#985)
primer/brand: .changeset/fair-ladybugs-bathe.md

---
'@primer/react-brand': patch
---

> The focus outline color of `SubNav` links can now be set using `var(--brand-color-focus);`

Reviewer: ```suggestion
The focus outline color of `SubNav` links is now set to `var(--brand-color-focus);`
```
